geoportal/geoportailv3_geoportal/static-ngeo/js/olcs/Lux3DManager.js

```diff
@@ -562,32 +558,21 @@ const exports = class extends ngeoOlcsManager {
   }
 
   restore_2D_layers_and_background() {
-    if (this.currentBgLayer !== undefined) {
-      this.backgroundLayerMgr_.set(this.map, this.currentBgLayer);
+    if (this.currentBgLayer !== undefined && this.currentBgLayer !== null) {
```

Comment:
Maybe `if (this.currentBgLayer)` would be strict enough?

Reply:
yes, good idea

Reply:
yes, good idea

Reply:
yes, good idea